src/components/Breed/ImagesGallery.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import classes from "./ImagesGallery.module.css";
```

Comment:
Try to use StyledComponents as everybody is using it and it will be good for you to get use to it. But, there is nothing wrong with this approach